python/ray/tune/ray_trial_executor.py

```diff
@@ -154,6 +154,12 @@ def __init__(self,
                  result_buffer_length: Optional[int] = None,
                  refresh_period: Optional[float] = None,
                  wait_for_placement_group: Optional[float] = None):
+        # TODO(xwjiang): Expose this through executor's ctor.
+        self._is_single_process = bool(os.environ.get("IS_SINGLE_PROCESS", 0))
```

Comment:
```suggestion
        self._is_single_process = bool(int(os.environ.get("IS_SINGLE_PROCESS", "0")))
```
Two things to note here:

- `os.environ` vars are always strings
- `bool("0") == True`, but `bool(int("0")) == bool(0) == False`